scripts/docker/entrypoint_ci.sh

```diff
@@ -204,6 +204,12 @@ function determine_airflow_to_use() {
         mkdir -p "${AIRFLOW_SOURCES}"/logs/
         mkdir -p "${AIRFLOW_SOURCES}"/tmp/
     else
+        if [[ ${USE_AIRFLOW_VERSION} =~ 2\.[7-8].* && ${TEST_TYPE} == "Providers[fab]" ]]; then
+            echo
+            echo "${COLOR_YELLOW}Skipping FAB tests on Airflow 2.7 and 2.8 because of FAB incompatibility with them${COLOR_RESET}"
+            echo
+            return
+        fi
```

Comment:
But this is not the only case we will encounter?
We have the information of what is the minimum airflow version for a provider I think we should reuse it
https://github.com/apache/airflow/blob/44800b2fde420dc551fec361a48a6305c3443977/dev/breeze/src/airflow_breeze/global_constants.py#L489-L508

Thus every time we have irregular min airflow version it would be picked automatically

Reply:
Later yes. There are some todos to do it 'properly' but this is a quick fix for Jed's problem while I am at CommunityOverCode